app/services/figgy_graphql.rb

```diff
@@ -0,0 +1,38 @@
+module FiggyGraphql
+  require "graphql/client"
+  require "graphql/client/http"
+  # Configure GraphQL endpoint using the basic HTTP network adapter.
+  HTTP = GraphQL::Client::HTTP.new("https://figgy.princeton.edu/graphql")
```

Comment:
I found the variable name a little confusing — is this an established convention I'm not familiar with? Or is there some other reason this is all caps, at the top level, but not an instance variable?

Reply:
Yeah, the graphql library wants everything to be in constants: https://github.com/github/graphql-client#configuration

Reply:
👍